src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.ts

```diff
@@ -81,6 +78,7 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
         private resultService: ResultService,
         private profileService: ProfileService,
         private programmingSubmissionService: ProgrammingSubmissionService,
+        private exportResultButtonComponent: ExerciseScoresExportButtonComponent,
```

Comment:
I didn't know that you can inject Angular components. Since you're only using it to use the `getResults` method, wouldn't it be better practice to extract that function into the `ResultService` and use that instead?

Reply:
I agree

Reply:
Good point :+1: 

https://github.com/ls1intum/Artemis/pull/3263/commits/e72002538fe248b8357fd77286669a3c48c2474a
Injection have now been removed and `getResults` method extracted into `ResultsService` as suggested

Thanks :slightly_smiling_face: